api/src/main/java/de/oliver/fancyholograms/api/events/HologramsLoadedEvent.java

```diff
@@ -0,0 +1,39 @@
+package de.oliver.fancyholograms.api.events;
+
+import com.google.common.collect.ImmutableList;
+import de.oliver.fancyholograms.api.HologramManager;
+import de.oliver.fancyholograms.api.hologram.Hologram;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public final class HologramsLoadedEvent extends Event {
+
+    private static final HandlerList handlerList = new HandlerList();
+
+    private final ImmutableList<Hologram> holograms;
+
+    public HologramsLoadedEvent(@NotNull final ImmutableList<Hologram> holograms) {
+        super(!Bukkit.isPrimaryThread());
```

Comment:
```suggestion
        super(true);
```

We ensure that certain events are fired async, I think this should be one of them, loading holograms should always be an async operation

Reply:
Yep alright thats probably a good idea for enforce async loading.